### ISSUE TITLE
Question regarding use of Microsoft.Data.Sqlite

### ISSUE THREAD

**Issue by manish2535** [dotnet/efcore#15797]
I am using the Microsoft.Data.Sqlite nuget package on my project. I just noticed that the github page says its been archived and moved to EntityFrameworkCore. 

Can anyone tell me what this means for people using Microsoft.Data.Sqlite package by answering these questions?

**1**. Can I continue using Microsoft.Data.Sqlite? 

**2**. If yes, will it continue receiving updates and development?

**3**. if No, do I need to install EntityFrameworkCore Nuget package and remove the Microsoft.Data.Sqlite? My project uses actual Raw SQL like below, will I still be able to use those? I dont see anything like this in the https://docs.microsoft.com/en-us/ef/core/querying/raw-sql page.

```
sqlSqliteConnection.Open();
String tableCommand = "CREATE TABLE IF NOT " +
    "EXISTS MyTable (Primary_Key INTEGER PRIMARY KEY, " +
    "Text_Entry NVARCHAR(2048) NULL)";

SqliteCommand createTable = new SqliteCommand(tableCommand, sqlSqliteConnection);
createTable.ExecuteReader();
sqlSqliteConnection.Close();
```

Thank you.

**Comment by bricelam** [comment]
We just combined a few repos. See https://github.com/aspnet/Announcements/issues/320. Microsoft.Data.Sqlite is otherwise unaffected. Issues are tracked here with the [area-adonet-sqlite](https://github.com/aspnet/EntityFrameworkCore/issues?q=label%3A%22area-adonet-sqlite%22+is%3Aopen) label. In fact, there are already several [resolved issues](https://github.com/aspnet/EntityFrameworkCore/issues?q=is%3Aissue+label%3Aarea-adonet-sqlite+is%3Aclosed+label%3Aclosed-fixed) in the 3.0 codebase.

**Comment by manish2535** [comment]
Thanks for the confirmation.
